src/modules/commander/commander.cpp

```diff
@@ -2698,19 +2698,20 @@ int commander_thread_main(int argc, char *argv[])
 			     internal_state.main_state == commander_state_s::MAIN_STATE_STAB ||
 			     internal_state.main_state == commander_state_s::MAIN_STATE_ALTCTL ||
 			     internal_state.main_state == commander_state_s::MAIN_STATE_POSCTL) &&
-			    ((status.rc_signal_lost && status_flags.gps_failure) ||
+			    ((status.rc_signal_lost /*&& status_flags.gps_failure*/) ||
```

Comment:
It's hard to guess how things like commander are intended to work with over 5000 lines of code in one module and no documentation of design intent...